webapps-schema/src/main/java/io/camunda/webapps/schema/entities/usermanagement/TenantEntity.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.webapps.schema.entities.usermanagement;
+
+import io.camunda.webapps.schema.entities.AbstractExporterEntity;
+
+public class TenantEntity extends AbstractExporterEntity<TenantEntity> {
+
+  public static final String DEFAULT_TENANT_IDENTIFIER = "<default>";
```

Comment:
When is this used vs `TenantOwned.DEFAULT_TENANT_IDENTIFIER`?

Reply:
Good question, as we already have the protocol as a dependency we could just use `TenantOwned.DEFAULT_TENANT_IDENTIFIER` it would remain consistent as that enum is also used elsewhere i.e. the RequestMapper class.

In fact the parent class also suggests this too :) https://github.com/camunda/camunda/blob/main/webapps-schema/src/main/java/io/camunda/webapps/schema/entities/AbstractExporterEntity.java#L20